doc/man1/openssl.pod

```diff
@@ -781,6 +781,257 @@ client.
 The input format for the extra certificate and key, respectively.
 See L<openssl(1)/Format Options> for details.
 
+=item B<-xchain_build>
+
+Specify whether the application should build the certificate chain to be
+provided to the server for the extra certificates via the B<-xkey>,
+B<-xcert>, and B<-xchain> options.
+
+=item B<-xcertform> B<DER>|B<PEM>, B<-xkeyform> B<DER>|B<PEM>
+
+The input format for the extra certifcate and key, respectively.
+See L<openssl(1)/Format Options> for details.
+
+=back
+
+=head2 Verification Options
+
+Many OpenSSL commands verify certificates. The details of how each
+command handles errors is documented on the specific command page.
+
+Verification is a complicated process, consisting of a number of separate
+steps that are detailed in the following paragraphs.
+
+First, a certificate chain is built up starting from the supplied certificate
+and ending in a root CA.  It is an error if the whole chain cannot be
+built up.  The chain is built up by looking up the issuers certificate
+of the current certificate.  If a certificate is found which is its own
+issuer it is assumed to be the root CA.
```

Comment:
Shouldn't it be better smth like

If an issuer certificate is a certificate itself, the certificate is assumed to be the root CA.

?

Reply:
I tried a slightly different wording, let me know what you think.